tools/container/docker-entrypoint.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Function to display messages with different severity levels
+# Usage: icinga2_log <severity> <message>
+icinga2_log() {
+    local severity="$1"
+    local message="$2"
+
+    local color=""
+    local reset=""
+
+    # Check if we are running in a terminal that supports colors,
+    # otherwise fallback to plain text output.
+    if [ -t 2 ]; then
+        reset="\033[0m"
+        # Set color codes based on severity
+        case "$severity" in
+            "information")
+                color="\033[1;32m" # Green
```

Comment:
```suggestion
                color="\033[32m" # Green
```

Reply:
Why? Bold not bold - both same thing for me.